.github/workflows/pull.yml

```diff
@@ -35,7 +35,6 @@ jobs:
           { config: "default", shard: 2, num_shards: 2, runner: "linux.2xlarge" },
           { config: "distributed", shard: 1, num_shards: 1, runner: "linux.2xlarge" },
           { config: "docs_test", shard: 1, num_shards: 1,  runner: "linux.2xlarge" },
-          { config: "backwards_compat", shard: 1, num_shards: 1, runner: "linux.2xlarge" },
```

Comment:
It would be better to comment out, rather than delete the line (and add a comment when to re-enabled)

Reply:
Let's just delete since it looks like github actions is having trouble triggering these workflows.

We can revert this PR when backwards compat is fixed instead.

We could also leave a comment at the top of this block instead as well

Reply:
This is a json blob, and json blobs don't allow comments